components/Home/Donate/index.jsx

```diff
@@ -180,6 +204,15 @@ export const DepositServiceDonation = () => {
             <Paragraph>{e.value}</Paragraph>
           </EpochStatus>
         ))}
+
+        <Button
+          type="primary"
+          loading={isCheckpointLoading}
+          disabled={isExpectedEndTimeInFuture}
+          onClick={onCheckpoint}
+        >
+          Checkpoint
+        </Button>
```

Comment:
We don't have a good docs setup on this site like Contribute, so I'd suggest a line of help text here:

`<Text type="secondary">New epochs must be manually triggered by community members</Text>`